plugins/modules/consul.py

```diff
@@ -78,76 +78,77 @@
     service_port:
         type: int
         description:
-          - the port on which the service is listening. Can optionally be supplied for
-            registration of a service, i.e. if I(service_name) or I(service_id) is set
+          - The port on which the service is listening. Can optionally be supplied for
+            registration of a service, i.e. if I(service_name) or I(service_id) is set.
     service_address:
         type: str
         description:
-          - the address to advertise that the service will be listening on.
+          - The address to advertise that the service will be listening on.
             This value will be passed as the I(address) parameter to Consul's
             C(/v1/agent/service/register) API method, so refer to the Consul API
             documentation for further details.
     tags:
         type: list
         elements: str
         description:
-          - tags that will be attached to the service registration.
+          - Tags that will be attached to the service registration.
     script:
         type: str
         description:
-          - the script/command that will be run periodically to check the health
-            of the service. Scripts require I(interval) and vice versa.
+          - The script/command that will be run periodically to check the health of the service.
+            Requires I(interval) to be provided.
     interval:
         type: str
         description:
-          - the interval at which the service check will be run. This is a number
-            with a s or m suffix to signify the units of seconds or minutes e.g
-            C(15s) or C(1m). If no suffix is supplied, m will be used by default e.g.
-            C(1) will be C(1m). Required if the I(script) parameter is specified.
+          - The interval at which the service check will be run.
+            This is a number with a C(s) or C(m) suffix to signify the units of seconds or minutes e.g C(15s) or C(1m).
+            If no suffix is supplied C(s) will be used by default, e.g. C(10) will be C(10s).
+            Required if one of the parameters I(script), I(http), or I(tcp) is specified.
```

Comment:
```suggestion
          - Required if one of the parameters I(script), I(http), or I(tcp) is specified.
```
I think it's better to make this a new paragraph.